x/oracle/module.go

```diff
@@ -329,13 +331,27 @@ func (am AppModule) OnRecvPacket(
 	}
 
 	source := types.IBCSource{SourceChannel: packet.DestinationChannel, SourcePort: packet.DestinationPort}
+
+	ms := ctx.MultiStore()
+	msCache := ms.CacheMultiStore()
+	if msCache.TracingEnabled() {
+		msCache = msCache.SetTracingContext(
+			sdk.TraceContext(
+				map[string]interface{}{
+					// TODO: Find a good key of packet (like tx hash for transaction)
+					"packet": fmt.Sprintf("%X", tmhash.Sum(packet.GetData())),
```

Comment:
How about using `ctx.TxBytes()`? Not sure whether the transaction exists.

```suggestion
					"packet": fmt.Sprintf("%X", tmhash.Sum(ctx.TxBytes())),
```

Reply:
I want to use packet specific because it apply on this packet not all tx.